testing/e2e/skr/package.json

```diff
@@ -28,6 +28,7 @@
   "dependencies": {
     "@kubernetes/client-node": "0.22.1",
     "axios": "^1.7.4",
+    "execa": "^9.4.0",
```

Comment:
<img width="632" alt="image" src="https://github.com/user-attachments/assets/a18bdfc7-8c24-4b3d-8cb3-5acf7dc04950">

Reply:
During dependabot update this dependence was not removed

Reply:
Since the last dependabot update was reverted this PR should be closed.

Reply:
That was not the missing dependency.